jaeger/charts/linkerd-jaeger/templates/namespace.yaml

```diff
@@ -7,3 +8,4 @@ metadata:
     linkerd.io/extension: jaeger
   annotations:
     linkerd.io/inject: enabled
+{{ end -}}
```

Comment:
Can you add a new line at the end of file?

Reply:
done